src/schema/objects/files.yaml

```
@@ -130,27 +130,29 @@ code:
   display_name: Code
   file_type: directory
   description: |
-    A directory in which to store any code used to generate the derivatives from the
-    source data.
+    A directory in which to store any code
+    (for example the one used to generate the derivatives from the raw data).
+    See the [relevant section](SPEC_ROOT/03-modality-agnostic-files.md#code)
+    for more information.
```

Two things.

1) Is `objects` the right place to put these user-friendly hints to render? I've generally thought of terms as trying to be clear with minimal reference to the spec.

2) Style-wise I think it's better to use more specific names in URL text when possible:

```suggestion
    See the [Code section](SPEC_ROOT/03-modality-agnostic-files.md#code)
    for more information.
```

---

> 1. Is `objects` the right place to put these user-friendly hints to render? I'  generally thought of terms as trying to be clear with minimal reference to the spec.

Yes I agree that this is suboptimal though I am even less of a fan of the alternative which is to have a macro call for each subdirectory.